packages/cspell/src/app/lint/lint.test.ts

```diff
@@ -24,6 +26,11 @@ const oc = <T>(obj: T) => expect.objectContaining(obj);
 const j = path.join;
 
 describe('Linter Validation Tests', () => {
+    beforeEach(() => {
```

Comment:
This could be `afterEach` and then the `delete` at the end of most of the tests isn't necessary.